runtime/vm/threadhelp.cpp

```diff
@@ -451,11 +457,18 @@ timeCompensationHelper(J9VMThread *vmThread, U_8 threadHelperType, omrthread_mon
 	if ((compensationMightBeRequired && J9_IS_CRIU_RESTORED(vm))
 		|| isThreadHaltedAtSingleThreadedMode
 	) {
-		/* There was a C/R and a compensation is required,
+		/* There was a C/R and a compensation might be required,
 		 * or this is a halted thread at single threaded mode.
 		 */
 		bool timeCompensationRequired = TRUE;
-		if (waitTimed) {
+		const I_8 timeout10ms = 10;
+		bool timedout = FALSE;
```

Comment:
This is C++ code: the legal constants for `bool` are `true` and `false` (`TRUE` and `FALSE` must undergo coercions).
This line should say `bool timedout = false;` and line 510 should say `timedout = true;`.
@JasonFengJ9 Can you please open a PR to correct this? @tajila FYI.

Reply:
Created https://github.com/eclipse-openj9/openj9/pull/16231